app/assets/stylesheets/objects/_side.scss

```diff
@@ -7,15 +7,17 @@
 }
 
 .app-side__actions {
-  @include govuk-responsive-margin(3, "bottom");
-
   .govuk-button {
     width: 100%;
     margin-bottom: govuk-spacing(3);
   }
 
-  .govuk-link {
-    margin-bottom: govuk-spacing(3);
```

Comment:
I get that this is a leftover from when we introduced the paragraph with `govuk-body govuk-!-margin-bottom-3` classes (with which I'm not very happy with) to wrap these links.

Reply:
Sure, I think the problem we have on the links is the margin only has an effect on the floated ones.